src/gluonnlp/data/utils.py

```diff
@@ -18,26 +18,25 @@
 # under the License.
 
 """Utility classes and functions. They help organize and keep statistics of datasets."""
-from __future__ import absolute_import
-from __future__ import print_function
+from __future__ import absolute_import, print_function
 
-__all__ = [
-    'Counter', 'count_tokens', 'concat_sequence', 'slice_sequence', 'train_valid_split',
-    'line_splitter', 'whitespace_splitter', 'Splitter'
-]
-
-import os
 import collections
-import zipfile
+import os
 import tarfile
-import numpy as np
+import zipfile
 
+import numpy as np
+import six
```

Comment:
six is not yet a dependency of mxnet. any way around it?

Reply:
removed